src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -444,5 +460,182 @@ internal static bool ContainsPathComponent(string path, string component, bool i
 
             return false;
         }
+
+        /// <summary>
+        /// Gets a path relative to a directory.
+        /// </summary>
+        public static string GetRelativePath(string directory, string fullPath)
```

Comment:
Yes, but we could easily copy their implementation until it is available everywhere. I am asking how ours differs and why? Is there something covered in one that is missing in the other (possibly some cross plat consideration that was missed) or is one more performant.